editoast/openapi.yaml

```diff
@@ -42,6 +42,62 @@ paths:
               schema: 
                 type: array
 
+  /layers/layer/{layer_slug}/mvt/{view_slug}/:
+    get:
+      summary: Mvt View Metadata
+      operationId: mvt_view_metadata_layer__layer_slug__mvt__view_slug___get
```

Comment:
I'm sure this can be made shorter

Reply:
It's not needed, this lines were autogenerated by fastapi.